a.css

```diff
@@ -120,3 +121,317 @@ a {
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.s3h3 {
+  position: static;
+  width: 327px;
+  height: 36px;
+  left: 0;
+  top: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.prj1 {
+  position: static;
+  width: 327px;
+  height: 147px;
+  left: 0;
+  top: 48px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.body2 {
+  background-color: #e5e5e5;
+  width: 343px;
+  margin-left: -24px;
+  margin-top: -36px;
+  padding: 40px;
+  height: fit-content;
+  padding-bottom: 100px;
+}
+
+.lan1 {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  justify-content: space-around;
+  margin-left: 0;
+}
+
+.q1 {
+  position: static;
+  width: 25px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.btn1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  position: static;
+  width: 124px;
+  height: 48px;
+  left: 0;
+  top: 259px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 12px 0;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
+
+.project2 {
+  background-image: url(image/proart.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  height: 450px;
+}
+
+.lan2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style-type: none;
+  justify-content: space-evenly;
+  padding-left: 10px;
+}
+
+.s4h1 {
+  font-family: Arial, Helvetica, sans-serif;
+  height: 81px;
+  width: 295px;
+  left: 16px;
+  top: 8px;
+  border-radius: nullpx;
+  margin-left: 16px;
+  padding-top: 80px;
+  color: #e5e5e5;
+}
+
+.p2 {
+  position: static;
+  width: 295px;
+  height: 87px;
+  left: 16px;
+  top: 101px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+  margin-left: 16px;
+}
+
+.btn2 {
+  position: absolute;
+  width: 343px;
+  background: #ff6b00;
+  height: 60px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
```

Comment:
- [ ] Let's remove the border from the project button by using `border: none;` so your button design matches the Figma design.

<img width="121" alt="image" src="https://user-images.githubusercontent.com/77190203/164861918-ea912d9a-47dd-494b-ae9a-fba165750a92.png">